### PR TITLE
#3 created sql functions for custom block

### PR DESCRIPTION
**Pull request by Libbna**
In this commit, deleted 2 extra files which was not required - controller.php and test.php.
Updated the sql file.
And created 2 functions for inserting block details and displaying them.

### CODE REVIEW

**Review by vbadkar — APPROVED** [review]
Looks good to me

File: custom_cms.sql [hunk 1]
```diff
@@ -24,6 +24,17 @@ SET time_zone = "+00:00";
 
 -- --------------------------------------------------------
 
+--
+-- Table structure for table `customBlock`
+--
+
+CREATE TABLE `customBlock` (
```
Comment (JayKandari) [inline custom_cms.sql:31]: How are we going to identify any unique block?

**Review by JayKandari — CHANGES_REQUESTED** [review]

File: views/templates/layout.html.twig [hunk 2]
```diff
@@ -16,17 +16,17 @@
...
 				</li>
 				<li>    
-					<a href="/custom_cms/contact" class="nav-link text-white">
+					<a href="/cms/contact" class="nav-link text-white">
 						Contacts
 					</a>
 				</li>
 				<li>    
-					<a href="/custom_cms/block" class="nav-link text-white">
+					<a href="/cms/block" class="nav-link text-white">
```
Comment (JayKandari) [inline views/templates/layout.html.twig:29]: Can we remove our dependence on the basepath  and instead just keep the actual URL ?

File: src/Models/Database.php [hunk 3]
```diff
@@ -33,4 +33,21 @@ public function insertUserDetails($name, $phone){
         $ans = $query->get_result();
         return $ans;
     }
+
+    // query for inserting block 
+    public function insertBlockDetails($title, $desc) {
+        $query = $this->conn->prepare("INSERT INTO customBlock(block_title, block_desc) VALUES('$title', '$desc')");
```
Comment (JayKandari) [inline src/Models/Database.php:39]: We are not correctly using the prepared statement by inserting variables inside a string. can we properly use the prepared statement and pass data variables correctly

**Review by JayKandari — CHANGES_REQUESTED** [review]

File: custom_cms.sql (hunk 1 above)
Comment (Libbna) [reply custom_cms.sql:31]: 9be20c63: I have added a unique id field in table through which we can identify any unique block.

**Pull request merged**